testing/src/main/java/org/oppia/android/testing/TextInputAction.kt

```diff
@@ -0,0 +1,39 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.view.View
+import androidx.annotation.StringRes
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+class TextInputAction @Inject constructor(
+  val context: Context
+) {
+  fun hasErrorText(@StringRes expectedErrorTextId: Int): Matcher<View> {
+    return object : TypeSafeMatcher<View>() {
```

Comment:
Can we make this specific for `TextInputLayout` as per this comment - https://github.com/oppia/oppia-android/pull/2721#discussion_r592893279

Reply:
![Screenshot from 2021-07-26 17-08-49](https://user-images.githubusercontent.com/76056229/126982938-9873d308-0388-4d34-9c3b-935d8d490c38.png)
@prayutsu is this correct?

Reply:
Yes, also rename the variable name from `view` to `textInputLayout`.

Reply:
![Screenshot from 2021-07-26 23-34-51](https://user-images.githubusercontent.com/76056229/127037105-d7291684-74db-4240-9a8d-3caf4aeea0e8.png)
Changing this causes this error on other files because we are passing TextInputLayout but View was expected.

Reply:
@prayutsu PTAL

Reply:
I think you may have to revert this then, but I suggest waiting for @BenHenning's thoughts on it.

Reply:
Done.